hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestHarness.java

```diff
@@ -225,6 +227,14 @@ protected void initMetaClient(HoodieTableType tableType) throws IOException {
     metaClient = HoodieTestUtils.init(hadoopConf, basePath, tableType);
   }
 
+  protected Properties getPropertiesForKeyGen() {
+    Properties properties = new Properties();
+    properties.put(HoodieTableConfig.HOODIE_POPULATE_META_COLUMNS.key(), "false");
```

Comment:
HoodieTestDataGenerator has these fields for commonly used schema and hence hardcoded it here.